### ISSUE TITLE
stage management layout - media management

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/mobilise#488]
in the media section of stage management, it is inconsistent that the media type is listed in the left-hand menu, while the uploader is a dropdown menu. they should both be dropdown menus, as they are both a way of filtering the media to find what you want. 

if the media type moves to being a dropdown menu next to the player name (between player name and the search box), then the "Scenes" menu item could become a sub-menu of "Media" (where currently the media types are), as this is essentially to do with the media on the stage.

thoughts, @vickis @elisadcg  ?

**Comment by vickis** [comment]
sure
makes sense :)

**Comment by nguyenhongphat0** [comment]
So each scene should have its own media? 🤔 Why we just not put all the media required for a performance in a stage, it will be easier to reuse media between multiple scenes, and easier to preload them when entering the stage.

**Comment by helenvarley** [comment]
yes sure, that's how it will work - we don't need to add media for scenes, only assign it once to the stage. & perhaps we don't need the Scenes to be in Stage Management at all, but as we've never had this feature before we need to play with it a bit to work out what we need.
in Stage Management, the Scenes interface would presumably show a list of scenes (if any have been created on that stage) and the media included in each scene; and presumably the possibility to edit the scene - e.g. remove media, or delete the scene. but this might not be such a useful thing in the Backstage, because you can't see how things would be arranged on the live stage. i guess you could add audio, or remove media, but you couldn't really add graphics other than a backdrop, because you need to be on the stage to position and size avatars or props.
we can delete scenes on stage already. do we need to be able to edit scenes? & if so could this be built into the on-stage tool? or is it really just as easy to delete and begin again?
